Resources/Prototypes/Entities/Objects/Specific/Mech/mechs.yml

```diff
@@ -116,4 +122,52 @@
         Blunt: 14
   - type: MovementSpeedModifier
     baseWalkSpeed: 2.25
-    baseSprintSpeed: 3.6
\ No newline at end of file
+    baseSprintSpeed: 3.6
+
+- type: entity
+  id: MechHAMTR
+  parent: BaseMech
+  name: HAMTR
+  description: An armored experimental Mech designed to be piloted by rodents.
+  components:
+  - type: FootstepModifier
+    footstepSoundCollection:
+      path: /Audio/Mecha/sound_mecha_powerloader_step.ogg
+  - type: Mech
+    baseState: hamtr
+    openState: hamtr-open
+    brokenState: hamtr-broken
+    pilotWhitelist:
+      tags:
+        - Hamster
+    mechToPilotDamageMultiplier: 0.75
+  - type: MeleeWeapon
+    hidden: true
+    attackRate: 0.9
+    damage:
+      types:
+        Blunt: 15
+  - type: MovementSpeedModifier
+    baseWalkSpeed: 2.25
+    baseSprintSpeed: 3.6
+  - type: Repairable
+    fuelCost: 15
+    doAfterDelay: 8
+
+- type: entity
+  id: MechHAMTRBattery
+  parent: MechHAMTR
+  suffix: Battery
+  components:
+  - type: Mech
+    baseState: hamtr
+    openState: hamtr-open
+    brokenState: hamtr-broken
+    startingBattery: PowerCellHigh
```

Comment:
if/when mech refactor merged replace this with just

```
components:
- type: ContainerFill
  containers:
    mech-battery-slot:
    - PowerCellHigh
```

Reply:
will do when it gets merged

Reply:
done